cypress/support/fragments/requests/requests.js

```diff
@@ -314,6 +326,12 @@ export default {
     cy.wait(2000);
   },
 
+  addNewTag(newTag) {
+    cy.do(cy.get('div#input-tag').click().wait(2000).type(newTag));
```

Comment:
why not interactor?

Reply:
@manowarqqq  Frankly speaking, I've tried Interactors and other ways but this component is laggy on Cypress and the only way I find to deal with it in a stable manner is this one.